egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -1478,7 +1478,7 @@ fi
 if ! "${skip_upload_hf}"; then
     if [ ${stage} -le 16 ] && [ ${stop_stage} -ge 16 ]; then
         [ -z "${hf_repo}" ] && \
-            log "ERROR: You need to setup the variable hf_repo with the name of the repository located at HuggingFace" && \
+            log "ERROR: You need to setup the variable hf_repo with the name of the repository located at HuggingFace, follow the following steps described here https://github.com/espnet/espnet/pull/4027/files#diff-b9f381af6d3ca8c14a68dca2a24de815830013fd69a3fe6a113e108c7aaf0d2dR1490-R1505" && \ 
```

Comment:
The link here is not the correct link to file, but the pull info. Can you fix it as well?

Reply:
Sure , updated with a permanent link

Reply:
Could you make it not stick to a specific commit?  It won't reflect future changes with a fixed commit.

Reply:
Hey in the latest commit the link is updated as https://github.com/espnet/espnet/blob/5cc31d748984c7332176ca31a541905c8709300a/CONTRIBUTING.md?plain=1#L68 , if this is not the permalink  could guide me how to get it ?

Reply:
The link can be `https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#132-espnet2-recipes·

Reply:
updated the link